src/hb-blob.cc

```diff
@@ -71,15 +71,49 @@ hb_blob_create (const char        *data,
 		hb_memory_mode_t   mode,
 		void              *user_data,
 		hb_destroy_func_t  destroy)
+{
+  if (!length)
+    return hb_blob_get_empty ();
```

Comment:
Do we need to call `destroy()` here like in the (old) code below?

Reply:
Riiiiight.